api/specs/webserver/openapi-catalog.yaml

```diff
@@ -85,14 +85,14 @@ paths:
         schema:
           title: Service key
           type: string
-          pattern: ^(simcore)/(services)/(comp|dynamic)(/[^\\s/]+)+$
+          pattern: ^(simcore)/(services)/(comp|dynamic|frontend)(/[\w/-]+)+$
```

Comment:
hmm sure about this one? did you check that file-picker never comes through?

Reply:
fixed